### PR TITLE
Update usecases

### PR DESCRIPTION
**Pull request by Steven-Carriger**
Updated the use case scenarios and added additional ones to be up to date with our ideas for the features in this sprint.

### CODE REVIEW

File: documentation/Use Case Scenarios/Completing Habit Scenario.txt [hunk 1]
```diff
@@ -2,9 +2,15 @@ Context: The user has loaded the application and has added a scenario
 Primary Flow:
 1.	The user selects a habit
 2.	The user selects the circle next to the habit
-3.	The habit is marked as complete 
-4.	The habit is sent to the bottom of the list with the other completed habits.
-5.	The user receives coins for successfully completing the task.
-6.	The coin amount increases by the coins gained.
+3.	The user is prompted to ensure they want to complete the habit
+4.	The user selects "yes" confirming they want the habit marked as complete
```
Comment (TakenouchiTR) [inline documentation/Use Case Scenarios/Completing Habit Scenario.txt:6]: The user should be able to check multiple habits at once, then mark them all as complete with a single prompt.

File: documentation/Use Case Scenarios/Registering Account Scenario.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Context: the user is a first time user
+Primary Flow:
+1.	The user loads the application
+2.	The user is taken to the login page
+3.	The user selects "register"
+4.	The user is taken to a registration page
```
Comment (TakenouchiTR) [inline documentation/Use Case Scenarios/Registering Account Scenario.txt:6]: We may or may not have to update this after our discussion, depending on what @bfreder1 wants to do.

**Review by TakenouchiTR — CHANGES_REQUESTED** [review]
Overall very good. I only have one hard change and one possible change, depending on what we decide to do with registration.

File: documentation/Use Case Scenarios/Completing Habit Scenario.txt [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
 1.	The user selects a habit
-2.	The user selects the circle next to the habit
-3.	The habit is marked as complete 
-4.	The habit is sent to the bottom of the list with the other completed habits.
-5.	The user receives coins for successfully completing the task.
-6.	The coin amount increases by the coins gained.
+2.	The user selects the check box next to the habits they want to complete
+3.	The user is prompted to ensure they want to complete the habits
+4.	The user selects "yes" confirming they want the habits marked as complete
+5.	The habits are marked as complete
```
Comment (TakenouchiTR) [inline documentation/Use Case Scenarios/Completing Habit Scenario.txt:7]: Check boxes should not bring up a confirmation box every time. Users should be able to checks multiple habits, then press a button to complete them. Once the button is pressed, it should ask if the user is sure.

**Review by TakenouchiTR — CHANGES_REQUESTED** [review]

**Review by TakenouchiTR — CHANGES_REQUESTED** [review]
All tabbing was removed entirely, when it should have just been fixed to be consistent.
There should also probably be a blank line between the primary and alternative flows, just to make the text easier to read.

**Review by MatthewThompson35 — APPROVED** [review]
Tabbing is now consistent.

**Pull request merged**